src/main/docker/disableWeakTlsAlgorithms.patch

```diff
@@ -0,0 +1,98 @@
+--- java.security
```

Comment:
We need a copyright header for this file.  What is the comment format for patch files?

Reply:
It is `#` btw so it should be like https://github.com/CAFapi/opensuse-base-image/blob/develop/src/main/docker/startup/startup.sh.

Reply:
License header added and custom mapping added so the file is checked by the license plugin.